### PR TITLE
Add changeService action to marathonActions

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds the changeService action to the marathon actions and makes changing a service possible.

Changes in this PR
- [x] The modal is handling the events.
- [x] Changing the text of the submit button to change configuration

image

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

**Comment by Poltergeist** [comment]
Restest this please

**Comment by Poltergeist** [comment]
Retest this please

**Comment by aldipower** [comment]
Der Hammer! 
:smile: `/aldipower`

File: src/js/components/modals/ServiceFormModal.js [hunk 1]
```diff
@@ -216,7 +246,7 @@ class ServiceFormModal extends mixin(StoreMixin) {
   }
 
   render() {
-    let title = 'Deploy new Serivce';
```
Comment (aldipower) [inline src/js/components/modals/ServiceFormModal.js]: "This was Croatian"

**Comment by aldipower** [comment]
This works! Great!!!

File: src/js/components/modals/ServiceFormModal.js [hunk 2]
```diff
@@ -116,14 +123,31 @@ class ServiceFormModal extends mixin(StoreMixin) {
...
+
   handleCancel() {
     this.props.onClose();
   }
 
   handleSubmit() {
+    let marathonAction = MarathonStore.createService;
+
+    if (this.props.isEdit) {
+      marathonAction = MarathonStore.changeService;
```
Comment (pierlo-upitup) [inline src/js/components/modals/ServiceFormModal.js]: Let's avoid calling this 'change'. It's an edit. I suggest `editService`.

Comment (pierlo-upitup) [inline src/js/components/modals/ServiceFormModal.js]: Also I'm not sure I'm a fan of the cached action. Seems pretty arbitrary.

File: src/js/events/MarathonActions.js [hunk 3]
```diff
@@ -61,6 +63,26 @@ module.exports = {
     });
   },
 
+  changeService: function (data) {
```
Comment (pierlo-upitup) [inline src/js/events/MarathonActions.js]: let's stick to `editService`

File: src/js/components/modals/ServiceFormModal.js (hunk 2 above)
Comment (aldipower) [inline src/js/components/modals/ServiceFormModal.js]: Guter Punkt.

**Comment by pierlo-upitup** [comment]
I get a `tracker.js:10 Warning: Failed propType: Invalid prop `startValue` of type `number` supplied to `FieldInput`, expected `string`. Check the render method of `FormControl`.` exception when i click edit.

**Comment by pierlo-upitup** [comment]
Also the sidebar went away 
image

**Comment by aldipower** [comment]
Are you sure this is related to this branch?
At least the sidebar issue I've noticed in `master` too yesterday.

**Comment by pierlo-upitup** [comment]
I am on `ph/feature/add-marathon-edit-action`

**Comment by pierlo-upitup** [comment]
Code LGTM. Merging as it goes into a feature branch where we can verify the above issues with greater confidence.

**Pull request merged**